dds/DCPS/RTPS/ICE/AgentImpl.cpp

```diff
@@ -88,9 +88,9 @@ int AgentImpl::handle_timeout(const ACE_Time_Value& a_now, const void* /*act*/)
 AgentImpl::AgentImpl() :
   ReactorInterceptor(TheServiceParticipant->reactor(), TheServiceParticipant->reactor_owner()),
   ncm_listener_added_(false),
-  remote_peer_reflexive_counter_(0)
-{
-}
+  remote_peer_reflexive_counter_(0) {
```

Comment:
Keep this line unmodified, `{` was fine where it was.

Reply:
I'll change it.